Content.Server/GameObjects/Components/GUI/InventoryComponent.cs

```diff
@@ -95,7 +95,8 @@ public ItemComponent GetSlotItem(Slots slot)
         }
         public T GetSlotItem<T>(Slots slot) where T : ItemComponent
         {
-            if (!SlotContainers.ContainsKey(slot))
+
+            if (!SlotContainers.TryGetValue(slot, out var containerSlot) || containerSlot == null)
```

Comment:
I can't comment on the right line, because github is dumb.

But now you're getting `containerSlot` out here, you should use that for line 103/104.

```cs
    var containedEntity = SlotContainers[slot].ContainedEntity;
```
to
```cs
    var containedEntity = containerSlot.ContainedEntity;
```